crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -17,8 +17,8 @@ use {
 ///
 /// * To get the global transform of an entity, you should get its [`GlobalTransform`].
 /// * For transform hierarchies to work correctly, you must have both a [`Transform`] and a [`GlobalTransform`].
-///   ~* You may use the [`TransformBundle`](crate::bundles::TransformBundle) to guarantee this.~
-///   * [`TransformBundle`](crate::bundles::TransformBundle) is now deprecated.
+///   * ~You may use the [`TransformBundle`](crate::bundles::TransformBundle) to guarantee this.~
+///     [`TransformBundle`](crate::bundles::TransformBundle) is now deprecated.
```

Comment:
The formatting for docs from the transform migration was scuffed so I decided to fixed it here